### PR TITLE
Unit Tests Cleanup (Fix For Issue #271)

### PR DESCRIPTION
**Pull request by thatzacdavis**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @thatzacdavis has signed the CLA. Thanks for the pull request!

File: Box.V2.Test/BoxCollaborationsManagerTest.cs [hunk 1]
```diff
@@ -2,30 +2,26 @@
...
-using System.Text;
 using System.Threading.Tasks;
 
 namespace Box.V2.Test
 {
     [TestClass]
     public class BoxCollaborationsManagerTest : BoxResourceManagerTest
     {
-        protected BoxCollaborationsManager _collaborationsManager;
+		private readonly BoxCollaborationsManager _collaborationsManager;
```
Comment (coolboy) [inline Box.V2.Test/BoxCollaborationsManagerTest.cs]: spacing, and please use 4 spaces to replace tab

File: Box.V2.Test/BoxCollectionsManagerTest.cs [hunk 2]
```diff
@@ -6,21 +6,18 @@
...
 using System.Threading.Tasks;
 
 namespace Box.V2.Test
 {
-
     [TestClass]
     public class BoxCollectionsManagerTest : BoxResourceManagerTest
     {
-        protected BoxCollectionsManager _collectionsManager;
+		private readonly BoxCollectionsManager _collectionsManager;
```
Comment (coolboy) [inline Box.V2.Test/BoxCollectionsManagerTest.cs]: ditto

File: Box.V2.Test/BoxCommentsManagerTest.cs [hunk 3]
```diff
@@ -3,22 +3,18 @@
...
-using System.Text;
 using System.Threading.Tasks;
 
 namespace Box.V2.Test
 {
     [TestClass]
     public class BoxCommentsManagerTest : BoxResourceManagerTest
     {
-        protected BoxCommentsManager _commentsManager;
+		private readonly BoxCommentsManager _commentsManager;
```
Comment (coolboy) [inline Box.V2.Test/BoxCommentsManagerTest.cs]: spacing

File: Box.V2.Test/BoxFoldersManagerTest.cs [hunk 4]
```diff
@@ -15,17 +15,17 @@ namespace Box.V2.Test
     [TestClass]
     public class BoxFoldersManagerTest : BoxResourceManagerTest
     {
-        protected BoxFoldersManager _foldersManager;
+		private readonly BoxFoldersManager _foldersManager;
```
Comment (coolboy) [inline Box.V2.Test/BoxFoldersManagerTest.cs]: spacing

File: Box.V2.Test/BoxGroupsManagerTest.cs [hunk 5]
```diff
@@ -1,31 +1,27 @@
...
-using Box.V2.Exceptions;
 
 namespace Box.V2.Test
 {
-   
     [TestClass]
     public class BoxGroupsManagerTest : BoxResourceManagerTest
     {
-        protected BoxGroupsManager _groupsManager;
+		private readonly BoxGroupsManager _groupsManager;
```
Comment (coolboy) [inline Box.V2.Test/BoxGroupsManagerTest.cs]: ditto

File: Box.V2.Test/BoxLegalHoldPoliciesManagerTest.cs [hunk 6]
```diff
@@ -12,11 +12,11 @@ namespace Box.V2.Test
     [TestClass]
     public class BoxLegalHoldPoliciesManagerTest : BoxResourceManagerTest
     {
-        protected BoxLegalHoldPoliciesManager _legalHoldPoliciesManager;
+		private readonly BoxLegalHoldPoliciesManager _legalHoldPoliciesManager;
```
Comment (coolboy) [inline Box.V2.Test/BoxLegalHoldPoliciesManagerTest.cs]: ditto and all other intances

File: Box.V2.Test/BoxServiceTest.cs [hunk 7]
```diff
@@ -16,12 +14,11 @@ namespace Box.V2.Test
     [TestClass]
     public class BoxServiceTest
     {
-
-        IBoxConverter _converter;
-        Mock<IRequestHandler> _handler;
-        IBoxService _service;
-        Mock<IBoxConfig> _boxConfig;
-        IAuthRepository _authRepository;
+	    private readonly IBoxConverter _converter;
```
Comment (coolboy) [inline Box.V2.Test/BoxServiceTest.cs]: spacing

File: Box.V2.Test/BoxResourceManagerTest.cs [hunk 8]
```diff
@@ -1,54 +1,46 @@
...
-        protected Uri _UserUri = new Uri(Constants.UserEndpointString);
-        protected Uri _InviteUri = new Uri(Constants.BoxApiUriString + Constants.InviteString);
+		protected Uri FoldersUri = new Uri(Constants.FoldersEndpointString);
+		protected Uri FilesUploadUri = new Uri(Constants.FilesUploadEndpointString);
+		protected Uri FilesUri = new Uri(Constants.FilesEndpointString);
+		protected Uri UserUri = new Uri(Constants.UserEndpointString);
+		protected Uri InviteUri = new Uri(Constants.BoxApiUriString + Constants.InviteString);
 
-        public BoxResourceManagerTest()
+	    protected BoxResourceManagerTest()
```
Comment (coolboy) [inline Box.V2.Test/BoxResourceManagerTest.cs]: spacing

File: Box.V2.Test/BoxMetadataManagerTest.cs [hunk 9]
```diff
@@ -3,23 +3,19 @@
...
-using System.Text;
 using System.Threading.Tasks;
 
 namespace Box.V2.Test
 {
     [TestClass]
     public class BoxMetadataManagerTest : BoxResourceManagerTest
     {
-        protected BoxMetadataManager _metadataManager;
+		private readonly BoxMetadataManager _metadataManager;
```
Comment (coolboy) [inline Box.V2.Test/BoxMetadataManagerTest.cs]: spacing

**Comment by thatzacdavis** [comment]
Sorry for the delay in fixing this up. Day job got in the way :)

**Comment by coolboy** [comment]
lgtm, thanks for the contribution @thatzacdavis

**Pull request merged**